core/src/main/java/org/apache/carbondata/core/util/CarbonTimeStatisticsFactory.java

```diff
@@ -52,12 +56,27 @@ public static LoadStatistics getLoadStatisticsInstance() {
     return LoadStatisticsInstance;
   }
 
-  private static DriverQueryStatisticsRecorder genQueryStatisticsRecorderInstance() {
-    return DriverQueryStatisticsRecorder.getInstance();
+  private static QueryStatisticsRecorder genQueryStatisticsRecorderInstance() {
```

Comment:
I think the function name of this function should be similar to `createDriverRecorder`